package.json

```diff
@@ -4,9 +4,10 @@
   "description": "Creating a clone of an item page of the brilliant earth engagement ring website with a team of four. This repo is for the customer reviews component.",
   "private": true,
   "scripts": {
+    "build": "webpack",
```

Comment:
Is this working for you? Doesn't look like you're providing it with any options or a config file. If you're just using this as a placeholder until you decide to tackle webpack, then disregard this comment.